awesomplete.css

```diff
@@ -1,4 +1,14 @@
 [hidden] { display: none; }
+[hiddena11y] {
+	border: 0;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
```

Comment:
Can you link to something about this hiding method? It’s so verbose... I guess the old `clip(0,0,0,0)` doesn’t work correctly?